operator/docs/examples/dataplane.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kit.k8s.sh/v1alpha1
+kind: DataPlane
+metadata:
+  name: example-nodes 
+spec: 
+  clusterName: example # Desired Cluster Name
```

Comment:
Let's be consistent on cluster vs control plane. Can we use CrossVersionObjectRef or something similar to support namespaces, kind, etc.